lib/graphql/queries.js

```diff
@@ -162,6 +162,18 @@ export const loggedInUserQuery = gql`
             country
             structured
           }
+          ... on Event {
+            paymentMethods {
+              type
+              service
+            }
+          }
+          ... on Project {
+            paymentMethods {
+              type
+              service
+            }
+          }
```

Comment:
In the spirit of https://github.com/opencollective/opencollective/issues/2126, we want to avoid fetching too many things in this `loggedInUserQuery` that is triggered very often.

The step payment is dynamic and has its own query to fetch payment methods, could we move it there? https://github.com/opencollective/opencollective-frontend/blob/247e741fdd5f899f2d60c5448e41cf57a2e5ec91/components/contribution-flow/StepPayment.js#L36

Reply:
I think it is possible. I'll check how often the internal payment method requirement is true vs. false and remove this.